### PR TITLE
fixes #53 : SEO Optimization

### LINKED ISSUE #53: Optimize the SEO
Find and fix the issues with the SEO compatibility of the website.

### PR DESCRIPTION
**Pull request by Oyedare**
# Description

Made changes in the code by adding meta tags and meta descriptions for SEO optimization.

Fixes #53 

## Motivation and Context
 This change was necessary for google search ranking

### CODE REVIEW

**Comment by sudip-mondal-2002** [comment]
Hi @Oyedare Can you please add the dependency on the app level package.json instead of the root level? Otherwise it'll go for production errors

File: package.json [hunk 1]
```diff
@@ -27,5 +27,8 @@
     "@azure/static-web-apps-cli": "^0.8.3",
     "azure-functions-core-tools": "^3.0.4484",
     "concurrently": "^7.1.0"
+  },
```
Comment (sudip-mondal-2002) [inline package.json:30]: Add the Dependency to the APP level package.json instead of root level package.json.

File: package.json [hunk 2]
```diff
@@ -27,5 +27,8 @@
...
+  "dependencies": {
+    "react-helmet": "^6.1.0"
```
Comment (sudip-mondal-2002) [inline package.json:32]: Add the Dependency to the APP level package.json instead of root level package.json.

Comment (sudip-mondal-2002) [reply package.json:32]: I think everything else looks good.

Comment (Oyedare) [reply package.json:32]: Okay

**Pull request merged**